File: src/Services/API/Domain/StoredProcedure/StoredProcedureParameter.cs
namespace API.Domain.StoredProcedure
{
    public class StoredProcedureParameter
{
    public string ParameterName { get; set; }
    public string Type { get; set; }
    public int Length { get; set; }
    public int Precision { get; set; }
    public int Scale { get; set; }
    public int ParameterOrder { get; set; }
    public string Collation { get; set; }
    public string ExtendedProperty { get; set; }
}

}

File: src/WebClient/mssql.client/src/app/procedure/model/StoredProcedureParameter.ts
// Represents a parameter for a stored procedure.
export class StoredProcedureParameter {
    parameterName: string;
    type: string;
    length: number;
    precision: number;
    scale: number;
    parameterOrder: number;
    collation: string;
    extendedProperty: string;
    isEditing: boolean;
    constructor(init?: Partial<StoredProcedureParameter>) {
        Object.assign(this, init);
    }
}
